graphjet-core/src/main/java/com/twitter/graphjet/hashing/ArrayBasedIntToIntArrayMap.java

```diff
@@ -72,6 +72,13 @@ public ReaderAccessibleInfo(
 
   private static final long DEFAULT_RETURN_VALUE = -1L;
 
+  private static final int INTEGER_TOP_TWO_BYTE_SHIFT = 1 << 16;
+  private static final int INTEGER_TOP_TWO_BYTE_OVERFLOW = 0x7fff;
+  private static final int FAVORITE_ACTION = 1;
+  private static final int RETWEET_ACTION = 2;
+  private static final int REPLY_ACTION = 3;
+  private static final int QUOTE_ACTION = 7;
```

Comment:
We should have a TODO item to consolidate the edge types into one file. Currently this is scattered into several places (tweet/TopSecondDegreeByCountForTweet, tweetfeature/TopSecondDegreeByCountForTweet, and here), which might pose problems in the future